templates/themes/modern/video_listen.html

```
@@ -158,7 +158,7 @@ <h3 class="text-lg font-bold mb-4 text-on-surface flex items-center gap-2">
                                     <img loading="lazy"
                                          class="w-10 h-10 rounded-full object-cover group-hover:ring-1 group-hover:ring-primary transition-all"
                                          src="{{ "{}@100w".format(rep['member']['avatar']) |pic }}"
-                                         alt="" />
+                                         alt="{{ v['title'] }}" />
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)